src/Tribe/Promoter/Observer.php

```diff
@@ -137,14 +138,34 @@ public function ticket_moved_type( $ticket_type_id, $destination_id, $source_id,
 		}
 	}
 
+	/**
+	 * Observer when an attendee is moved from a post to another and notify Promoter about changes on both events
+	 *
+	 * @since TBD
+	 *
+	 * @param int $ticket_id the ticket which has been moved
+	 * @param int $source_ticket_type_id the ticket type it belonged to originally
+	 * @param int $target_ticket_type_id the ticket type it now belongs to
+	 * @param int $source_event_id the event/post which the ticket originally belonged to
+	 * @param int $target_event_id the event/post which the ticket now belongs to
+	 * @param int $instigator_id the user who initiated the change
+	 */
+	public function ticket_moved( $ticket_id, $source_ticket_type_id, $target_ticket_type_id, $source_event_id, $target_event_id, $instigator_id ) {
+		$this->notify( $source_event_id );
+		// Prevent to send the same response twice if the ID's are the same.
+		if ( $source_event_id !== $target_event_id ) {
+			$this->notify( $target_event_id );
+		}
+	}
+
 	/**
 	 * Notify the connector of changes when the event was deleted
 	 *
 	 * @since 4.10.9
 	 *
 	 * @param $post_id
 	 */
-	public function delete_post( $post_id ) {
+	public function on_event_deleted( $post_id ) {
```

Comment:
Since we are modifying the method name here we should create the old one as a deprecated method and point to this one. Also, update both of their Docblocks to reflect the change.